testing/utilities.py

```diff
@@ -235,46 +322,40 @@ def test_get_inputs(self):
         '''
         
         inputs = requests.get('{0}/inputs'.format(self.url)).json()
-        self.assertEqual(len(inputs), len(self.inputs_ref))
-        for inp in inputs:
-            self.assertTrue(inp in self.inputs_ref)
-            self.assertTrue(inputs[inp]['Unit'] == self.inputs_ref[inp]['Unit'])
-            self.assertTrue(inputs[inp]['Description'] == self.inputs_ref[inp]['Description'])
-            self.assertTrue(inputs[inp]['Minimum'] == self.inputs_ref[inp]['Minimum'])
-            self.assertTrue(inputs[inp]['Maximum'] == self.inputs_ref[inp]['Maximum'])
+        ref_filepath = os.path.join(get_root_path(), 'testing', 'references', self.name, 'get_inputs.txt')
```

Comment:
This reference file is missing for each test case.

Reply:
Good catch.  This is because the ``.gitignore`` has a line *.txt, and I did not notice that those files did not get staged.  Changing to .json as suggested above fixed this problem and these files have been committed in https://github.com/ibpsa/project1-boptest/pull/189/commits/b76cfb448de10d6b64a91e7f497c6de96709156c.